### PR TITLE
SAK-31115 - elFinder not longer working for browsing My Workspace

### PR DESCRIPTION
**Pull request by mcarropellicer**
elFinder not longer working for browsing My Workspace

### CODE REVIEW

**Comment by mcarropellicer** [comment]
I suggest to move the the changes from "build.min.js" to "ckeditor.launch.js", should be something like:

collectionId = collectionId.replace('/user/','/user/~');

**Comment by juanjmerono** [comment]
I think is a great suggestion, modifying a minified js is not a good idea.

**Comment by mcarropellicer** [comment]
@juanjmerono Indeed, fix moved to "ckeditor.launch.js".

File: reference/library/src/webapp/editor/ckeditor.launch.js [hunk 1]
```diff
@@ -84,6 +84,9 @@ sakai.editor.editors.ckeditor.launch = function(targetId, config, w, h) {
         var elfinderUrl = '/library/editor/elfinder/sakai/elfinder.' + elfinderBuild +
             '.html?connector=elfinder-connector/elfinder-servlet/connector';
 
+        //SAK-31115 Permission error while getting resources from workspace
```
Comment (ern) [inline reference/library/src/webapp/editor/ckeditor.launch.js]: Remove jira reference

File: textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/SakaiFsService.java [hunk 2]
```diff
@@ -164,6 +166,12 @@ public String getVolumeId(FsVolume volume) {
 
 	public FsVolume[] getVolumes() {
 		List<Site> sites  = siteService.getSites(ACCESS, null, null, null, null, null);
+		//SAK-31115 elFinder not longer working for browsing My Workspace
```
Comment (ern) [inline textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/SakaiFsService.java]: another comment to remove

**Comment by mcarropellicer** [comment]
@ern jira comments removed.

**Comment by mcarropellicer** [comment]
I've followed the comments, is it ready to merge?

**Pull request merged**